core/peer/config_test.go

```diff
@@ -354,6 +354,59 @@ func TestGlobalConfigDefault(t *testing.T) {
 	assert.Equal(t, expectedConfig, coreConfig)
 }
 
+func TestPropagateEnvironment(t *testing.T) {
+	defer viper.Reset()
+	viper.Set("peer.address", "localhost:8080")
+	viper.Set("chaincode.externalBuilders", &[]ExternalBuilder{
+		{
+			Name:        "testName",
+			Environment: []string{"KEY=VALUE"},
+			Path:        "/testPath",
+		},
+		{
+			Name:                 "testName",
+			PropagateEnvironment: []string{"KEY=VALUE"},
+			Path:                 "/testPath",
+		},
+		{
+			Name:                 "testName",
+			Environment:          []string{"KEY=VALUE"},
+			PropagateEnvironment: []string{"KEY=VALUE"},
```

Comment:
Shouldn't these be different so we can determine which took precedence?

Reply:
done